.vscode/settings.json

```diff
@@ -15,6 +15,7 @@
     "editor.codeActionsOnSave": {
         "source.fixAll.eslint": "always"
     },
+    "files.eol": "\r\n",
```

Comment:
Do you think it's doable using prettier instead?
Maybe the current prettier configuration should be update to do that.
Also, did you run "yarn lint" and/or "yarn lint-fix"? Maybe it already fixes that issue